conda_forge_tick/migrators/qt_to_qt_main.py

```diff
@@ -0,0 +1,17 @@
+from conda_forge_tick.migrators.core import Replacement
+
+
+class QtQtMainMigrator(Replacement):
```

Comment:
`Replacement` is a full migrator and not a mini-migrator. We want a mini-migrator to go with the pinning update

Reply:
ok. understood.